src/Resources/Applications/Requests/UpdateBasicDetails.php

```
@@ -33,6 +33,10 @@ public function createDtoFromResponse(Response $response): Application
     {
         $data = $response->json();
 
+        if ( ! is_array($data['app']['id'])) {
```

PHPStan (2.0.3)

Cannot access offset 'id' on mixed.


(offsetAccess.nonOffsetAccessible)